website/index.html

```diff
@@ -1,11 +1,22 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="manifest" href="static/manifest.json">
```

Comment:
Please put this tag below `<meta charset="utf-8">`.

`<meta charset="utf-8">` should always come first because the document may have to be re-parsed when this tag is encountered.